kodilla-testing/src/test/java/com/kodilla/testing/forum/statistics/CalculateStatisticsTestSuite.java

```diff
@@ -0,0 +1,192 @@
+package com.kodilla.testing.forum.statistics;
+
+import org.junit.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class CalculateStatisticsTestSuite {
+
+    private static int testCounter = 0;
+
+    @BeforeClass
+    public static void beforeAllTests() {
+
+        System.out.println("Start Testing: ");
+    }
+
+    @AfterClass
+    public static void afterAllTests() {
+
+        System.out.println("The end of all tests.");
+    }
+
+    @Before
+    public void beforeEveryTest() {
+        testCounter++;
+        System.out.println("Test nr " + testCounter);
+    }
+    @After
+    public void afterEveryTest() {
+
+        System.out.println("End of test nr " + testCounter);
+    }
+
+    @Test
+    public void zeroPostTest() {
+        //Given
+        Statistics statisticsMock = mock(Statistics.class);
+        int postCounter = 0;
+        List<String> users = new ArrayList<>();
+        users.add("Jola");
+        users.add("Ola");
```

Comment:
Mozna to zapisac ktorej `List<String> users = Arrays.asList("Jola", "Ola");`

Reply:
O, dzięki, szukałem tego w internecie ale nie mogłem tego znaleźć